react-function-components/src/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import * as ReactDOM from 'react-dom';
+
+class CustomButton extends React.Component {
+  render() {
+    return <button>Click Me</button>;
+  }
+}
```

Comment:
this is actually a class component and not a functional component, I would switch it to a functional one

Reply:
Changed to a function